poll/public/js/poll.js

```diff
@@ -18,7 +18,23 @@ function PollUtil (runtime, element, pollType) {
         }
 
         // Make gettext available in Handlebars templates
-        Handlebars.registerHelper('i18n', function(str) { return gettext(str); });
+        Handlebars.registerHelper('i18n', function(str, options) {
+            options = options.hash;
+            str = gettext(str);
+            if (options) {
+                str = str.supplant(options)
```

Comment:
@asadiqbal08 Yeah, to do that sort of string properly, you need to add support for [ngettext](https://docs.djangoproject.com/en/1.9/topics/i18n/translation/#pluralization). A basic JavaScript implementation of `ngettext` is already available in the edX LMS/CMS.

To write this in handlebars, you would do:
```
{{sprintf (i18n_ngettext "Results gathered from %s respondent" "Results gathered from %s respondent" count) count}}
```

Then you need to define a new handlebars helper (just like the `i18n` one), called `i18n_ngettext`, which accepts three arguments and passes them to the global `ngettext` function.

I believe that `sprintf` is a standard handlebars tag, and the syntax above is what handlebars uses for using two helpers at once (first ngettext, then sprintf), but I've never used any of this stuff before myself so you may have to do some more investigation.